### PR TITLE
[Mentor 'Sign Up' pop-up] the text "Already have a Space2Study account? Log In" doesn't correspond the requirements #72

### LINKED ISSUE #72: [Mentor 'Sign Up' pop-up] the text "Already have a Space2Study account? Log In" doesn't correspond the requirements
**Defect Type:** UI/Usability
**Severity:** Trivial
**Priority:** Low
**Reproducible:** Always
**Environment:** Windows 10 21H2, Chrome 110.0.5481.178

**Description:** 
The text "Already have a Space2Study account? Log In" doesn't correspond the requirements

**Preconditions:**
The web-page on the client is opened, the button "Become a Tutor" is clicked and the "Tutor Sign Up" pop-up appeared.

**Steps to reproduce:**
Take a look at the text below the "Sign Up with Google" button.

**Actual result:**
The text under "Sign Up with Google" button is "Already have a tutor account? Login!" and it is not corresponding the mockup.
image

**Expected result:**
The text under "Sign Up with Google" button is "Already have a Space2Study account? Log In!" and is matching to mockup.

### PR DESCRIPTION
**Pull request by docia**
edited "Log In" button. 

before: 
Screenshot 2024-04-26 at 21 46 20

after: 
Screenshot 2024-04-26 at 21 46 33

### CODE REVIEW

**Review by BodiaPutiak — APPROVED** [review]

**Review by ohanovdmytro — APPROVED** [review]

File: src/constants/translations/en/signup.json
Comment (Olenka-Hryk) [inline src/constants/translations/en/signup.json:1]: The current implementation doesn't match the mockup precisely. According to the requirements, the text on the button should read: "Already have a Space2Study account? Log In!"

Please, update the button text to match the specified requirement.

File: src/constants/translations/en/signup.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "googleButton": "Sign up with Google",
   "continue": "or continue",
   "haveAccount": "Already have a tutor account?",
```
Comment (Olenka-Hryk) [inline src/constants/translations/en/signup.json]:
```suggestion
  "haveAccount": "Already have a Space2Study account?",
```

File: tests/unit/components/loader/Loader.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import Loader from '~/components/loader/Loader.jsx'
```
Comment (Olenka-Hryk) [inline tests/unit/components/loader/Loader.spec.jsx]: These changes are already included in this PR #111. Could you please revert then or change the base branch from develop to #111 ?

File: src/constants/translations/en/signup.json (hunk 1 above)
Comment (docia) [reply src/constants/translations/en/signup.json]: thanks for the edit!

**Review by Olenka-Hryk — APPROVED** [review]

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**